group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -306,6 +308,22 @@ public CoordinatorResult<OffsetCommitResponseData, Record> commitOffset(
         return offsetMetadataManager.commitOffset(context, request);
     }
 
+    /**
+     * Handles a OffsetCommit request.
+     *
+     * @param context The request context.
+     * @param request The actual OffsetCommit request.
+     *
+     * @return A Result containing the OffsetCommitResponse response and
```

Comment:
"A result containing the offset commit response", we don't want to use names in the java doc, just the description right?

Reply:
fixed the javadocs to LeaveGroup. The other APIs also are using the actual API name (SyncGroup/JoinGroup...) so I will keep LeaveGroup as is.